test/auth.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const chai = require('chai')
+const expect = chai.expect
+const mojang = require('../')
+
+describe('mojang.auth()', () => {
+
+  // TODO implement test scenario with valid credentials 
```

Comment:
For example, before you make the `mojang.auth(...)` calls, setup `nock('https://authserver.mojang.com')` mocks.